### ISSUE TITLE
MPD binding does mix up status information from different players

### ISSUE THREAD

**Issue by alexf2015** [openhab/openhab#3060]
I configured following items:
/* MPD */
String 	    SZCurrentTrack          "Current track [%s]"                                      (g2OG_Schlafzimmer)                 { mpd="TITLE:musicsz:tracktitle" }
String      SZCurrentArtist         "Current artist [%s]"                                     (g2OG_Schlafzimmer)                 { mpd="ARTIST:musicsz:trackartist" }
String 	    WZCurrentTrack          "Current track [%s]"                                      (g2OG_Wohnzimmer)                   { mpd="TITLE:musicwz:tracktitle" }
String      WZCurrentArtist         "Current artist [%s]"                                     (g2OG_Wohnzimmer)                   { mpd="ARTIST:musicwz:trackartist" }

and following mpd configuration:
mpd:musicsz.host=192.168.8.46
mpd:musicsz.port=6600
mpd:musicwz.host=192.168.8.37
mpd:musicwz.port=6600

The problem is that whenever the currenttrack or currentartist of one of the players changes the labels of both players are updated with those values.

**Comment by steve-bate** [comment]
Will you please move further discussion of this question to the [community forum] (https://community.openhab.org)? Thanks.

**Comment by teichsta** [comment]
@steve-bate i would consider this issue a bug report.Could @patrickse @petrklus or @mgbowman could have a look into this, please?

**Comment by patrickse** [comment]
@steve-bate Please enable debug logging for the package org.openhab.binding.mpd. Inside your logs should be an entry starting with ``Current song``. That implies that the song changed. The line should be followed by to paramters. The first one should be the playerid and the second should be the song title.

What happens if a song changes on __musicsz__. Which entry will show up in your logfile.

**Comment by steve-bate** [comment]
@patrickse To be clear, @alexf2015 reported this issue. I was just reviewing the status and I don't currently use the MPD binding.

**Comment by patrickse** [comment]
Oh sorry. I am a bit in a hurry ;)...

**Comment by alexf2015** [comment]
a little bit late but here is the debug output:

```
2015-12-09 19:32:46.240 [DEBUG] [.o.b.mpd.internal.MpdActivator] - MPD binding has been started.
2015-12-09 19:32:46.471 [DEBUG] [inding.mpd.internal.MpdBinding] - MPD binding connecting to players
2015-12-09 19:32:46.491 [DEBUG] [inding.mpd.internal.MpdBinding] - MPD binding connecting to players
2015-12-09 19:32:46.637 [DEBUG] [inding.mpd.internal.MpdBinding] - Connected to player 'musicsz' with config MPD [host=192.168.8.47, port=6600, password=null]
2015-12-09 19:32:46.712 [DEBUG] [inding.mpd.internal.MpdBinding] - Connected to player 'musicwz' with config MPD [host=192.168.8.37, port=6600, password=null]
2015-12-09 19:32:46.726 [DEBUG] [inding.mpd.internal.MpdBinding] - Scheduled a daily MPD Reconnect of all MPDs
2015-12-09 19:32:49.466 [DEBUG] [inding.mpd.internal.MpdBinding] - Volume on musicwz changed to 85
2015-12-09 19:32:49.472 [DEBUG] [inding.mpd.internal.MpdBinding] - Volume on musicsz changed to 22
2015-12-09 19:33:34.801 [DEBUG] [inding.mpd.internal.MpdBinding] - Current song musicsz: Der Super-Papagei
2015-12-09 19:33:34.806 [DEBUG] [inding.mpd.internal.MpdBinding] - Updated title: WZCurrentTrack Der Super-Papagei
2015-12-09 19:33:34.808 [DEBUG] [inding.mpd.internal.MpdBinding] - Updated title: SZCurrentTrack Der Super-Papagei
2015-12-09 19:33:34.810 [DEBUG] [inding.mpd.internal.MpdBinding] - Updated artist: WZCurrentArtist, Die 3 Fragezeichen
2015-12-09 19:33:34.813 [DEBUG] [inding.mpd.internal.MpdBinding] - Updated artist: SZCurrentArtist, Die 3 Fragezeichen
2015-12-09 19:33:51.449 [DEBUG] [inding.mpd.internal.MpdBinding] - Play state of 'musicsz' changed
```

**Comment by empi89** [comment]
I have the same issue with volume status of different MPD instances. Changes from one instance also affect the display-value other one.

**Comment by empi89** [comment]
This is my workaround for this: https://github.com/empi89/mqtt2mpd

**Comment by mczerski** [comment]
LOL, this bug can be fixed in 5 minutes of coding ... but still opened since 2015 ...

**Comment by 9037568** [comment]
Fixed by PR #4713.
